invenio_rdm_records/resources.py

```diff
@@ -32,3 +36,37 @@ class BibliographicRecordResource(RecordResource):
     """Record resource."""
 
     default_config = BibliographicRecordResourceConfig
+
+
+class BibliographicDraftResourceConfig(DraftResourceConfig):
+    """Bibliographic Record resource config."""
+
+    list_route = "/rdm-records/<pid_value>/draft"
+    response_handlers = {
+        "application/json": RecordResponse(
+            DraftJSONSerializer(schema=BibliographicRecordDraftSchemaJSONV1)
```

Comment:
I fully agree, the problem is that we need to agree in the full datamodel. Most importantly on how to make required vs not required fields. At the moment the draft is requiring all (since it uses the normal MetadataSchema). But yes.